module/icp/algs/sha2/sha2_generic.c

```diff
@@ -226,7 +228,8 @@ static void sha512_generic(uint64_t state[8], const void *data, size_t num_blks)
 	}
 }
 
-static void sha256_update(sha256_ctx *ctx, const uint8_t *data, size_t len)
+static void
+_sha256_update(sha256_ctx *ctx, const uint8_t *data, size_t len)
```

Comment:
Sure the functions are static, and it should be fine from compiler/linker perspective, but I wonder if more specific name could be useful for some debugging/instrumentation?